src/scripts/main.js

```diff
@@ -1,3 +1,46 @@
 'use strict';
 
-// write code here
+const employeeListItems = document.querySelectorAll('li');
+const employeeListArray = Array.from(employeeListItems);
```

Comment:
```suggestion
const employeeListArray = [...employeeListItems];
```
you can use spread operator to create an array. I belive this is shorter)